### PR TITLE
Merge from release/3.3 to fix broken docs build

### PR DESCRIPTION
**Pull request by awholegunch**
To resolve conflicts:

- cdap-docs/admin-manual/build.sh (chose version in release/3.3)
- cdap-explore-jdbc/src/main/java/co/cask/cdap/explore/jdbc/ExploreDriver.java (chose version in develop)

Need to run build to check...

Running as: http://builds.cask.co/browse/CDAP-DUT3636-1

### CODE REVIEW

**Comment by anwar6953** [comment]
LGTM

**Comment by anwar6953** [comment]
Build failed due to 2 flaky (log-related) test cases. Rerunning: http://builds.cask.co/browse/CDAP-DUT3636-3

**Comment by awholegunch** [comment]
Need to re-run build to check...

Running as: http://builds.cask.co/browse/CDAP-DUT3636-4

File: cdap-docs/vars [hunk 1]
```diff
@@ -22,3 +22,6 @@ MANUALS="introduction developers-manual cdap-apps admin-manual integrations exam
 
 # This needs to be explicitly set as Git has trouble identifying it.
 GIT_BRANCH_PARENT="develop"
+
+# Used by cdap-apps for Hydrator documentation
+GIT_BRANCH_CDAP_HYDRATOR="release/1.2"
```
Comment (anwar6953) [inline cdap-docs/vars]: this needs to change, right?

Comment (awholegunch) [inline cdap-docs/vars]: Fixed.

**Comment by anwar6953** [comment]
LGTM

**Pull request merged**